### PR TITLE
Add types for form contact details

### PR DESCRIPTION
**Pull request by davidjamesstone**

### CODE REVIEW

**Review by alexluckett — APPROVED** [review]

File: model/src/form/form-metadata/index.ts [hunk 1]
```diff
@@ -41,6 +41,33 @@ export const privacyNoticeUrlSchema = Joi.string()
...
+  })
+  .trim()
+  .required()
+export const onlineLinkTextSchema = Joi.string().trim().required()
+export const onlineSchema = Joi.object().keys({
+  url: onlineLinkUrlSchema,
+  text: onlineLinkTextSchema
+})
+
+export const contactSchema = Joi.object().keys({
```
Comment (colinrotherham) [inline model/src/form/form-metadata/index.ts:65]: Should we pass the expected types into Joi?

It's not the most advanced feature, but it'll shout at us if we miss new keys in future etc

```suggestion
export const contactSchema = Joi.object<FormMetadataContact>().keys({
```

Comment (davidjamesstone) [reply model/src/form/form-metadata/index.ts]: Resolved 68b3c9ec2f7f8c6b4775cf740ee46f4716d14792

Comment (colinrotherham) [reply model/src/form/form-metadata/index.ts]: Happy to leave the other objects for now?

Comment (davidjamesstone) [reply model/src/form/form-metadata/index.ts]: For the child schema? c22fd5e999a68152ec20ecd73350eca0245a0232

Comment (colinrotherham) [reply model/src/form/form-metadata/index.ts]: Cheers, think you've got them all now

**Review by colinrotherham — APPROVED** [review]

**Pull request merged**